hack/app-sre/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+# This kustomize configuration deploys what is required for the hive-operator
+# *without* OLM.
+# Except:
+# It does not contain a HiveConfig.
+
+# Adds namespace to all resources.
+namespace: hive
```

Comment:
you shouldn't add the namespace to the resources, this is determined by app-interface (think of a possible future hive deployment in multiple namespaces on the same cluster)